Modules/GameState.cs

```diff
@@ -107,6 +107,10 @@ public void SetMainRole(CustomRoles role)
         {
             countTypes = CountTypes.Crew;
         }
+        if (pc.Is(CustomRoles.Narc))
```

Comment:
is narc a main role or a sub role?
oc.is(role) is used to check main role i think

Reply:
> is narc a main role or a sub role? oc.is(role) is used to check main role i think

I thought it was used to check subrole :/